app/views/docs/_install_homebrew.html.md

```diff
@@ -1 +1 @@
-    brew update && brew install asciinema
+brew install asciinema
```

Comment:
Ah, I see. I think if you merge now, it should come through without the indention. As in, you edited the commit.